ssr-playground/src/entry-client.js

```diff
@@ -0,0 +1,6 @@
+import { createElement } from "lwc";
+import Parent from "x/parent";
+
+const elm = createElement("x-parent", { is: Parent });
+elm.tabIndex = 1;
```

Comment:
Why set the tabindex here? Also typically a11y audits will warn about 1) tabindex greater than 0, and 2) tabindex without `aria-label`.